doc/tools.md

```diff
@@ -103,3 +103,49 @@ Go incluye una herramienta para ejecutar los tests:
 [go test](https://pkg.go.dev/cmd/go/internal/test). Es la herramienta por
 defecto y forma parte del estado del arte de Go. Por tanto, no es necesario
 incluir otra herramienta para ejecutar los tests.
+
+## Imagen de Docker
+
+### Criterios de elección
+
+- **Huella de memoria pequeña**: la imagen debe ser lo más pequeña posible para
+  disminuir el tiempo necesario para descargarla y ejecutarla.
+
+- **Herramientas necesarias para el desarrollo**: la imagen debe incluir el
+  mayor número posible de herramientas necesarias para mi proyecto, en este caso
+  un proyecto de Go.
+
+- **Soporte**: la imagen debe estar mantenida y actualizada.
+
+### Opciones a considerar
+
+- [golang](https://hub.docker.com/_/golang): imagen oficial de Docker para Go y
+  es mantenida por la comunidad de Docker. Es una imagen muy popular, con más de
+  mil millones de descargas. Al ser la imagen oficial de Docker, está muy bien
+  mantenida y actualizada. Tiene varias variantes disponibles:
+
+  - `golang`: es la imagen por defecto, basado en Debian.
+
+  - `golang:alpine`: basado en Alpine Linux, una distribución muy ligera de
+    Linux que tiene un tamaño de 5MB. Esta imagen sin embargo tiene un número
+    mínimo de herramientas, dejando que sea el usuario el que instale las
+    herramientas que necesite.
+
+- [alpine](https://hub.docker.com/_/alpine): imagen de Alpine Linux, una
+  distribución muy ligera de Linux que tiene un tamaño de 5MB. Esta imagen sin
+  embargo, tiene un número mínimo de herramientas, dejando que sea el usuario el
+  que instale las herramientas que necesite.
+
+- Imágenes basadas en distribuciones de Linux como Debian, Ubuntu, CentOS,
+  Fedora: estas imágenes tienen un tamaño mayor que las imágenes basadas en
+  Alpine Linux, pero incluyen más herramientas por defecto. Sin embargo, no
+  tienen instaladas las herramientas de Go.
+
+### Decisión para el proyecto
```

Comment:
¿Por qué os limitáis a las imágenes oficiales? De Go precisamente hay un montón más...

Reply:
LLevo un rato buscando otras imágenes de Go y no encuentro ninguna "no oficial" que se actualice habitualmente. Están las opciones de servicios de integración continua como CircleCI, pero están pensadas para trabajar con sus servicios.

Reply:
¿Y estás rechazando de plano que vayas a usar CircleCI en el siguiente?
Por otro lado, simplemente usando la etiqueta golang estoy viendo al menos 3 imágenes de "verified publishers" que se actualizan periódicamente.

Reply:
> LLevo un rato buscando otras imágenes de Go y no encuentro ninguna "no oficial" que se actualice habitualmente. Están las opciones de servicios de integración continua como CircleCI, pero están pensadas para trabajar con sus servicios.

Con no oficiales se refiere a que no sean del propio equipo de go, pero sí de usuarios verificados. Te dejo algunas interesantes que encontré en su momento por aquí si te sirven de ayuda:
- https://hub.docker.com/r/bitnami/golang
- https://hub.docker.com/r/okteto/golang (Consulta el repo de git asociado, es bastante curioso)
Después hay alguna más de usuarios con sponsor y bastante interesantes también.